instrumentation/afl-compiler-rt.o.c

```diff
@@ -516,12 +516,23 @@ static void __afl_map_shm(void) {
 
     }
```

Comment:
No further else? Is that as you wanted it?

Reply:
yes I don't think we should set 
__afl_map_size to MAP_INITIAL_SIZE;
if __afl_final_loc is already equivalent to the map size

https://github.com/AFLplusplus/AFLplusplus/issues/1528#issuecomment-1253318827
here in your comment it says `__afl_map_size 2097152` (MAP_INITIAL_SIZE) but it is wrong isn't it?